mlnode/test/test_client.py

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+from iotdb.mlnode.client import client_manager
+
+
+def test_client_manager():
+    assert client_manager.borrow_data_node_client()
+    assert client_manager.borrow_config_node_client()
```

Comment:
How will the test codes work if there isn' a working IoTDB? Maybe try IoTDBContainer.

Reply:
Currently, the namespace of mlnode and iotdb package is the same, making IoTDBContainer not avaible